src/IO/ConsoleIO.php

```diff
@@ -92,6 +92,22 @@ public function writeError($messages, $newline = true)
         $this->doWrite($messages, $newline, true);
     }
 
+    /**
+     * @return InputInterface
+     */
+    public function getInput()
```

Comment:
Why did you add these getters? They are not being used at the moment, so I would not make them available at this point.